Makefile

```diff
@@ -19,7 +19,13 @@ run: all
 build-sass: $(SASS_SRC)
 	$(SASS_COMPILER) $(MAIN_FILE) $(OUTPUT_FILE)
 
+build-docker-image:
+	docker build -t 2016-site .
+
+docker:
```

Comment:
Building the image takes some time (even with caching, it takes around 15 seconds) and a rebuild is only required every time the Dockerfile is changed hence why there is no dependency. Setting the Dockerfile as a target doesn't work as make always thinks it's up to date.